network-api/networkapi/buyersguide/views.py

```diff
@@ -154,6 +154,20 @@ def why_view(request):
     })
 
 
+@enforce_en_locale
+def contact_view(request):
+    key = 'categories'
+    categories = cache.get(key)
+
+    if not categories:
+        categories = BuyersGuideProductCategory.objects.all()
+        cache.set(key, categories, 86400)
+
+    return render(request, 'about/contact.html', {
```

Comment:
it feels like what we really need here is a fallback pattern in the `urls.py` for catching home, about, contact, etc. instead of dedidated views for every single page. Filed as https://github.com/mozilla/foundation.mozilla.org/issues/2610